elements/lisk-validator/src/validation.ts

```diff
@@ -242,3 +242,18 @@ export const validateFee = (data: string): boolean =>
 	isNumberString(data) &&
 	isGreaterThanZero(new BigNum(data)) &&
 	!isGreaterThanMaxTransactionAmount(new BigNum(data));
+
+export const isCsv = (data: string): boolean => {
```

Comment:
Can we test this function in `elements/lisk-validator/test/validation.ts`?

Reply:
Added test for `isCsv` resolving.